### PR TITLE
Add years to datamodel

### PR DESCRIPTION
**Pull request by s1182276**
This PR adds 
- Separate Years that can be linked to SchoolYears and can be used for business logic
- IsActive flag to SchoolYear for use with business logic

### CODE REVIEW

File: Mvc/Controllers/SchoolYearController.cs [hunk 1]
```diff
@@ -90,15 +110,18 @@ public async Task<ActionResult> CopySchoolYear(SchoolYearViewModel syvm)
...
-                Id = syvm.Id,
-                Name = syvm.Name,
-                SchoolModules = modules != null ? modules : null
-            };
+            //var schoolYearDto = new SchoolYearDto()
+            //{
+            //    Id = syvm.SchoolYear.Id,
+            //    Name = syvm.SchoolYear.Name,
+            //    SchoolModules = modules != null ? modules : null
+            //};
```
Comment (sisimaile) [inline Mvc/Controllers/SchoolYearController.cs:118]: Dit kan weg?

**Review by sisimaile — COMMENTED** [review]
Approved

File: Mvc/Controllers/SchoolYearController.cs (hunk 1 above)
Comment (s1182276) [reply Mvc/Controllers/SchoolYearController.cs:118]: Goed gespot, die was ik vergeten idd.

**Review by sisimaile — COMMENTED** [review]
Toppie Joppie

**Comment by sisimaile** [comment]
Toppie Joppie x2

**Comment by s1182276** [comment]
> Toppie Joppie x2

Drie keer scheepsrecht? :D

**Review by sisimaile — APPROVED** [review]

**Pull request merged**